tutorials/04_RunningREMINDandMAgPIE.md

```diff
@@ -136,11 +136,13 @@ Other, optional columns allow you to make a run start only after another has fin
    - `path_gdx`, `path_gdx_ref`, `path_gdx_refpolicycost`, `path_gdx_carbonprice`, `path_gdx_bau`, : Override these same settings in REMIND's `scenario_config`, see [`03_RunningBundleOfRuns`](./03_RunningBundleOfRuns.md) for a detailed explanation.
       - You can set these switches either to the full path of a `fulldata.gdx` file or simply to the name of another scenario in the file (without the "C_"!). So if you want a certain scenario (say `NDC`) to use as starting point the results of a `Base` scenario, you can simply set `path_gdx` to `Base` and it will automatically locate the appropriate `fulldata.gdx` in `Base`, for example `path_remind/C_Base-rem-5/fulldata.gdx` if 5 iterations where requested and you run in "serial" mode.
```

Comment:
great you updated the tutorial. may you also delete the reference to the `serial` mode here (basically, the last half sentence can be deleted)